### PR TITLE
[ABC-475] add padding to help box

### PR DESCRIPTION
**Pull request by arejayelle**
## Purpose of this PR

**Ticket/Jira #:**ABC-475

Fix a couple of things on the Helpbox:
- Bottom padding issue
- Length issue when inspector is wide.

image
image
image

## Testing

**Functional Testing status:** Manual tests.

**Performance Testing status:** None

## Overall Product Risks

**Complexity:** Low
**Minimal** / Low / Medium / High

**Halo Effect:** Minimal
**Minimal** / Low / Medium / High
## Documentation & UX Writing

| User facing text to review |  | Details |
| :--- | :--- | :--- |
| User interface | N |  |
| Public API docs | N |  |
| Changelog | N |  |

| Documentation halo effect |  | Jira link |
| :--- | :--- | :--- |
| User manual | Y |  |
| Other docs | N |  |

## Additional information

**Note to reviewers:**

**Reminder:**

- [ ] Add entry in CHANGELOG.md

### CODE REVIEW

**Comment by thomas-tu** [comment]
Icon looks weird on my monitor:
image

**Comment by thomas-tu** [comment]
Warning box has a fixed length. When the inspector is wide, the warning box doesn't increase in size.
image

**Comment by arejayelle** [comment]
> Icon looks weird on my monitor: [image]

OK so i looked into this and apparently there's just something wrong with the icon itself? I Tried to see if I was using the wrong code or size but no. they're all weird. I couldn't find any version of the warning icon that didn't have the weird bumps which is unfortunate.

**Review by cguer — COMMENTED** [review]
Do we use the same icon file for the Recorder package? It looks like it's rendering better in the Recorder window 🤔 

REC: warning_recorder
ABC: warning_alembic

**Comment by thomas-tu** [comment]
@cguer I've fixed a couple of things. UI should look good now as shown in the description.

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2023.3.0b3 (Windows only)

File: com.unity.formats.alembic/Editor/Misc/UIHelper.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                }
+                return s_HelpBoxStyle;
+            }
+        }
+
+        /// <summary>Draw a help box with a button.</summary>
+        /// <param name="message">The message, unlocalized.</param>
+        /// <param name="buttonLabel">The button text.</param>
+        /// <param name="action">When the user clicks the button, Unity performs this action.</param>
+        internal static void HelpBoxWithAction(string message, MessageType messageType, string buttonLabel, Action action)
```
Comment (SanaJal) [inline com.unity.formats.alembic/Editor/Misc/UIHelper.cs:35]: small detail : `MessageType messageType` doesn t have param name entry in the summary

**Review by SanaJal — APPROVED** [review]
LGTM ! tested on 2023.3 on Windows

File: com.unity.formats.alembic/Editor/Misc/UIHelper.cs (hunk 1 above)
Comment (thomas-tu) [reply com.unity.formats.alembic/Editor/Misc/UIHelper.cs:36]: Thanks, here: [b0ba5c8](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/577/commits/b0ba5c84a88ce8ee305355ae11d375cd459a1ae5)

**Pull request merged**